scripts/infra/cloud-instance.sh

```diff
@@ -263,6 +262,16 @@ ibm__ssh() {
     ssh -o "StrictHostKeyChecking no" -i "$IBM_KEY_LOCATION" "$user_name"@"$PUBLIC_DNS" "$@"
 }
 
+ibm__scp() {
+    handle_help_and_instance_name_opts "$@"
+    shift $((OPTIND-1))
+
+    local user_name
+    user_name=$(instance_user_name "ibm")
+    ibm_calculate_instance_public_dns
+    scp -o "StrictHostKeyChecking no" -i "$IBM_KEY_LOCATION" "$@" "$user_name"@"$PUBLIC_DNS":
```

Comment:
Why is this logic different?  Sure there is a good reason but I am not following from the syntax.

Reply:
Hmmm I don't follow - the logic matches what's in the AWS function, just using the respective IBM variables/helper funcs

Reply:
Sorry, I was looking at the wrong line.

Reply:
No worries :+1: